### ISSUE TITLE
project

### ISSUE THREAD

**Issue by shahr8888** [bearpelican/midi_generator#29]
Can someone please implement this project in my local host? I can easily pay you.

**Comment by Bogula** [comment]
Same here, I would pay for a working version

**Comment by asigalov61** [comment]
@shahr8888 @Bogula Hey guys,

I am also interested in developing/implementing music frontend so I was wondering if you want to connect and chat?

Maybe together we can implement it somehow. 

Let me know.

Alex

**Comment by alexwlloyd** [comment]
Yes i am working on getting this back up and running. Any success so far? I just gotta get the midi data base he had - one fo the links he gave is broken

**Comment by asigalov61** [comment]
@alexwlloyd Nope. I was not able to make it work. I was thinking of adopting the UI/Interface for one of my models (check out my repos on my profile). They work but need UI/front-end. I can help with hosting but I have no idea how to do the UI/interface.

**Comment by alexwlloyd** [comment]
@asigalov61 ok. I don't have much time right now, but I'll see what I can get up, UI wiser then I'll get back to you.  Of course @bearpelican , if you wanna chime in and give us some support? We can offer to compensate you financially.
